bundler/spec/runtime/inline_spec.rb

```diff
@@ -344,15 +344,25 @@ def confirm(msg, newline = nil)
 
   it "skips platform warnings" do
     bundle "config set --local force_ruby_platform true"
+    script <<-RUBY
+      gemfile(true) do
+        source "#{file_uri_for(gem_repo1)}"
+        gem "rack", platform: :jruby
+      end
+    RUBY
 
+    expect(err_without_deprecations).to be_empty
+  end
+
+  it "shows platform warnings" do
```

Comment:
Could you move the new spec to `spec/other/major_deprecation_spec.rb`? I like to keep all deprecations in one place so that I can go through all of them when we are about to release the next major.

Also, you need to skip this spec on bundler 3.0. In `spec/other/major_deprecation_spec.rb` you can see other specs that do this, normally we leave it as pending and I will address the 3.0 behavior before releasing that.

Reply:
Sure, I can do this.